vernac/g_vernac.mlg

```diff
@@ -218,12 +218,62 @@ GRAMMAR EXTEND Gram
           { VernacRegister(g, RegisterCoqlib quid) }
       | IDENT "Register"; IDENT "Inline"; g = global ->
           { VernacRegister(g, RegisterInline) }
+      | IDENT "Register"; IDENT "Inductive"; g = global; "as";
+          r = register_ind_token ->
+          { VernacRegister(g, RegisterInductive r) }
+      | IDENT "Primitive"; id = identref; ":="; r = register_token ->
+          { VernacPrimitive(id, r) }
       | IDENT "Universe"; l = LIST1 identref -> { VernacUniverse l }
       | IDENT "Universes"; l = LIST1 identref -> { VernacUniverse l }
       | IDENT "Constraint"; l = LIST1 univ_constraint SEP "," -> { VernacConstraint l }
   ] ]
   ;
 
+  register_ind_token:
+     [ [ "ind_bool" -> { CPrimitives.PIT_bool }
+       | "ind_carry" -> { CPrimitives.PIT_carry }
+       | "ind_pair" -> { CPrimitives.PIT_pair }
+       | "ind_cmp" -> { CPrimitives.PIT_cmp }
+       | "ind_eq"  -> { CPrimitives.PIT_eq } ] ]
```

Comment:
Isn't this making all these strings into keywords?

Reply:
Consider if it would be cleaner to take an ident and look it up in a separately defined associative list.

Reply:
Refactored in ec34d916660631fbba457e9893e4061815297d6d to reuse the syntax of coqlib.